From 9b38b97d1c6232e23f67bebb1ab8713dcdfd3763 Mon Sep 17 00:00:00 2001
From: insilications <boboniboni@gmail.com>
Date: Thu, 13 Aug 2020 07:53:14 -0300
Subject: [PATCH] Remove copyright check

---
 CMakeLists.txt | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6392d2b..d3d704c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -83,17 +83,6 @@ endif()
 
 find_host_package(PythonInterp 3 REQUIRED)
 
-add_custom_target(check-copyright ALL
-  ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/utils/add_copyright.py
-  --check
-  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-  COMMENT "Check copyright")
-
-add_custom_target(add-copyright
-  ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/utils/add_copyright.py
-  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-  COMMENT "Add copyright")
-
 if(MSVC)
   option(SHADERC_ENABLE_SHARED_CRT
           "Use the shared CRT instead of the static CRT"
-- 
2.27.0

