From a775a007fa2e9b61ee17ffcb4b1e6b271220af21 Mon Sep 17 00:00:00 2001
From: insilications <boboniboni@gmail.com>
Date: Sat, 29 Aug 2020 07:06:12 -0300
Subject: [PATCH] Remove copyright check

---
 CMakeLists.txt | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6392d2b..81fc720 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -83,16 +83,6 @@ endif()
 
 find_host_package(PythonInterp 3 REQUIRED)
 
-add_custom_target(check-copyright ALL
-  ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/utils/add_copyright.py
-  --check
-  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-  COMMENT "Check copyright")
-
-add_custom_target(add-copyright
-  ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/utils/add_copyright.py
-  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-  COMMENT "Add copyright")
 
 if(MSVC)
   option(SHADERC_ENABLE_SHARED_CRT
-- 
2.27.0

